be/src/runtime/vectorized/time_types.cpp

```diff
@@ -16,17 +16,17 @@ const uint64_t LOG_10_INT[] = {1,         10,         100,         1000,
 
 struct JulianToDateEntry {
     // 14 bits
-    uint16_t year;
+    uint16_t year : 14;
```

Comment:
theoretically, it will make the cache smaller. The packed size is 4bytes, and the original size is 6 bytes.